src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -142,11 +144,9 @@
             (:parsed-text content))))
 
 (defn quoted-message
-  [{:keys [message-id chat-id]} pin?]
-  (let [quoted-message (get @(re-frame/subscribe [:chats/chat-messages chat-id])
```

Comment:
Removing the subscription to `messages`, this only works if the message is in the view of the user (we fetch last 20 by default).